php-practice/section6.php

```diff
@@ -223,7 +270,18 @@ function printErrors(array $errors) {
  *
  * Now make the SAME printErrors function with a foreach loop!
  */
-
-function printErrorsAlternate(array $errors) {
-  // YOUR CODE HERE
+/**I'm going to need your help. I got this to work but I'm not sure why I didn't get the "string conversion error" I normally get.
+*/
+function printErrorsAlternate(array $altErrors) {
+  foreach($altErrors as $value) {
```

Comment:
Just as a side note, I would still call the parameter `$errors`.

Code inside of functions is isolated from code inside of other functions. So when you're inside `printErrorsAlternate`, ignore what any other variables are called. Call them what makes the most sense in the context of this function.